### DISCUSSION TITLE
[Feature request] Option to confirm save changes back to the currently connected database

### DISCUSSION THREAD

**Discussion by otykier** [TabularEditor/TabularEditor3#1035] [Ideas]
### Discussed in https://github.com/TabularEditor/TabularEditor3/discussions/921

<div type='discussions-op-text'>

<sup>Originally posted by **GrimnebulinAhab** June 20, 2023</sup>
It would be nice to have an option confirm changes before automatically saving back to the connected database.</div>

**Comment by DBojsen** [comment]
Hi @GrimnebulinAhab

I have added this to our backlog.  No ETA yet.
